FirebaseRemoteConfig/Sources/RCNConfigRealtime.m

```diff
@@ -384,24 +385,29 @@ - (void)fetchLatestConfig:(NSInteger)remainingAttempts targetVersion:(NSInteger)
     [strongSelf->_configFetch
         fetchConfigWithExpirationDuration:0
                         completionHandler:^(FIRRemoteConfigFetchStatus status, NSError *error) {
-                          NSLog(@"Fetching new config");
+                          FIRLogInfo(kFIRLoggerRemoteConfig, @"I-RCN000015",
+                                     @"Attempting to fetch new config.");
```

Comment:
based on today's feedback I wonder if some of these should be debug logs, or if we need them at all